styles.scss

```diff
@@ -508,6 +529,23 @@ blockquote {
   }
 }
 
+.NewVersionAlert {
+  position: fixed;
+  background-color: orange;
```

Comment:
Not sure about that orange. I could totally see using `$success` here (a shade of green), because a newer version available is very positive news. Alternatively, `$info` is more neutral.